contracts/interfaces/IAMMWrapper.sol

```diff
@@ -34,6 +44,19 @@ interface IAMMWrapper is IStrategyBase {
         bool relayed;
     }
 
+    /// @notice Trade with AMM
+    /// @param _makerAddress The AMM maker address
+    /// @param _fromAssetAddress The input assest of the trade
+    /// @param _toAssetAddress The output assest of the trade
+    /// @param _takerAssetAmount The input amount of the trade
+    /// @param _makerAssetAmount The output amount of the trade
+    /// @param _feeFactor The factor used to calculate fee, only listed relayer can set it effectively
+    /// @param _spender The address of the user
```

Comment:
Shall we rename `_spender` to `_userAddr` to align with he naming in implementation?

Reply:
Goot catch, fixed in 326d081